### PR TITLE
Feature/component handling

### PR DESCRIPTION
**Pull request by dcooney**
### Link

https://nextjs-wordpress-starter-a8ozji3n2.vercel.app/blog/test-post

### Description
This PR is to get base paragraph and heading blocks added.

- Adds Paragraph block handling.
- Adds Heading block handling.
- Adds component for handling blocks and helper functions.
- Adding WDS logo to header (🌮 )

### Screenshot

image

### Verification

How will a stakeholder test this?

1. View link on [vercel](https://nextjs-wordpress-starter-a8ozji3n2.vercel.app/blog/test-post)
2. View code

### CODE REVIEW

File: components/molecules/Blocks/Blocks.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import displayBlock from '@/functions/displayBlock'
+
+/**
+ * Render the Blocks component.
+ *
+ * @param {array} blocks An array of blocks.
```
Comment (gregrickaby) [inline components/molecules/Blocks/Blocks.js:8]: This docblock needs some ❤️ 

```suggestion
 * @author WebDevStudios
 * @param {object} props            The component attributes as props.
 * @param {array} props.blocks  An array of blocks.
```

File: components/molecules/Blocks/Blocks.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+ * @return {Element}     The Blocks component.
+ */
+export default function Blocks({blocks}) {
+  return (
+    <>
+      {
+        // If there are blocks, loop over and display.
+        !!blocks &&
```
Comment (gregrickaby) [inline components/molecules/Blocks/Blocks.js:16]: This multi-line conditional be shortened to a single line:

```js
!!blocks?.length &&
```

File: functions/createMarkup.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+/**
+ * Handle markup that contains HTML.
+ *
+ * @param {Array} props Array of JSX Objects.
```
Comment (gregrickaby) [inline functions/createMarkup.js:4]: This doc block is missing both @author and @return comments. Furthermore, your code editor _should_ be catching these.

**Review by gregrickaby — CHANGES_REQUESTED** [review]

File: functions/createMarkup.js (hunk 3 above)
Comment (dcooney) [reply functions/createMarkup.js:4]: VSCode isn't catching these. I'll have to investigate.

**Review by gregrickaby — APPROVED** [review]

**Pull request merged**